mysql-test/suite/versioning/r/replace.result

```diff
@@ -8,10 +8,47 @@ period for system_time(row_start, row_end)
 ) with system versioning;
 insert t values (1, 2);
 replace t values (1, 3);
-select *, current_row(row_end) as current from t for system_time all order by x;
-id	x	current
-1	2	0
-1	3	1
+replace t values (1, 3);
+select *, current_row(row_end) as current, count(row_start) = 1 as unique_start,
+row_start != 0 as start_nonzero
+from t for system_time all
+group by row_start order by x, row_end;
+id	x	current	unique_start	start_nonzero
+1	2	0	1	1
+1	3	0	1	1
+1	3	1	1	1
+# MDEV-15990: REPLACE on a precise-versioned table
+# returns duplicate key error (ER_DUP_ENTRY)
+replace t values (1, 3), (1, 30), (1, 40);
+select *, current_row(row_end) as current, row_start != 0 as start_nonzero
+from t for system_time all order by x, row_end;
+id	x	current	start_nonzero
+1	2	0	1
+1	3	0	1
+1	3	0	1
+1	40	1	1
+select count(row_start) as empty from t for system_time all
+group by row_start having count(row_start) > 1;
```

Comment:
Can you explain what you did you want to check with this query?

Reply:
In this test case we insert/modify one row per transaction. This means, that each `row_start` value occurs only once. 
Actually, for now it just turns into zero, but it's also possible to get other values by slightly playing with code, and it's not covered by tests.

On the other hand, it's not directly related to bug, yes. So it could be replaced with simple nonzero check, in scope of jira task